src/com/egs/arsensargsyan/task2/User.java

```diff
@@ -0,0 +1,29 @@
+package task2;
+
+public class User {
+
+    private String name;
```

Comment:
use have to use some filed as a unique identifier. either some id, or username ? and check each time before adding user into a list. there can't be duplicate users